src/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  posts: Post[];
+};
+
+const PostsList: React.FC<Props> = ({ posts }) => (
+  <ul>
+    {posts.map((post) => (
+      <li>
+        <b>{post.title}</b>
+        <div>{post.body}</div>
```

Comment:
i would like make assumption that `body` is some text, so its better to use `<p/> or <article/>`